README.md

```diff
@@ -2,21 +2,19 @@
 
 The GeoLambda project provides public Docker images and AWS Lambda Layers containing common geospatial native libraries. GeoLambda contains the libraries PROJ.5, GEOS, GeoTIFF, HDF4/5, SZIP, NetCDF, OpenJPEG, WEBP, ZSTD, and GDAL. For some applications you may wish to minimize the size of the libraries by excluding unused libraries, or you may wish to add other libraries. In this case this repository can be used as a template to create your own Docker image or Lambda Layer following the instructions in this README.
 
-This repository also contains examples and pre-built images for specific runtimes.
+This repository also contains additional images and layers for specific runtimes. Using them as a Layer assumes the use of the basee GeoLambda layer.
 
 - [Python](python/README.md)
 
 ## Usage
 
-While GeoLambda was initially intended for AWS Lambda they are also useful as base geospatial Docker images. The version of GDAL included for different versions of GeoLambda is shown here:
+While GeoLambda was initially intended for AWS Lambda it is also useful as a base geospatial Docker image. For detailed info on what is included in each image, see the Dockerfile for that version or the [CHANGELOG](CHANGELOG.md). A version summary is provided here:
 
-| geolambda | GDAL  |
-| --------- | ----  |
-| 1.0.0     | 2.3.1 |
-| 1.1.0     | 2.4.1 |
-| 1.2.0     | 2.4.2 |
-
-To get the version numbers of all installed packages see the [CHANGELOG](CHANGELOG.md) or the Dockerfile for the specific version.
+| geolambda | GDAL  | Notes |
+| --------- | ----  | |
+| 1.0.0     | 2.3.1 | |
+| 1.1.0     | 2.4.1 | |
+| 1.2.0     | 2.4.2 | Separate Python (3.7.4) image and Lambda Layer added |
```

Comment:
This table don't seems to render correctly in Markdown

Reply:
fixed